### PR TITLE
Used goimports to fix import order

### PR DESCRIPTION
**Pull request by muesli**
Fixed/sorted import order to match goimports

### CODE REVIEW

File: plugins/rest/rest.go [hunk 1]
```diff
@@ -10,12 +10,13 @@ import (
 	"context"
 	"encoding/json"
 	"errors"
-	"github.com/open-policy-agent/opa/internal/version"
 	"io"
 	"net/http"
 	"reflect"
 	"strings"
 
+	"github.com/open-policy-agent/opa/internal/version"
```
Comment (srenatus) [inline plugins/rest/rest.go]: Still a bit odd here

Comment (muesli) [reply plugins/rest/rest.go]: Ah, you mean we should group them with the other open-policy-agent imports below? Happy to take care of that!

Comment (srenatus) [reply plugins/rest/rest.go]: Yup that's what I meant. There are a few more of these below, though...

Comment (muesli) [reply plugins/rest/rest.go]: Hope I caught them all!

**Comment by patrick-east** [comment]
Thanks for the fixes! To prevent the imports from being mangled again in the future can you add `goimport` as part of the build targets with this change? I think it should just be a matter of updating the `./build/go-fmt.sh` script to use `goimports` instead of `gofmt` directly.

**Comment by muesli** [comment]
@patrick-east Sure thing. Done and done!

File: build/check-fmt.sh [hunk 2]
```diff
@@ -8,7 +8,7 @@ function opa::check_fmt() {
     exit_code=0
     for pkg in $(opa::go_packages); do
         for file in $(opa::go_files_in_package $pkg); do
-            __diff=$(gofmt -d $file | tee >(cat - >&5))
+            __diff=$(goimports -d $file | tee >(cat - >&5))
```
Comment (srenatus) [inline build/check-fmt.sh]: Does this need `-local github.com/open-policy-agent/opa`, or do I remember something that is no longer up-to-date? From my local `goimports --help`:
```
  -local string
        put imports beginning with this string after 3rd-party packages; comma-separated list
```

(Apparently, it's not breaking stuff right now, but I believe that without the `-local` flag, this check might be too weak?)

Comment (patrick-east) [reply build/check-fmt.sh]: Agreed, I think we might want to have the check (and changes for this PR to fix existing issues) be from something like `goimports -local github.com/open-policy-agent/opa -format-only -d ..`

**Comment by patrick-east** [comment]
@muesli did you have a chance to look into the latest issues @srenatus and myself brought up?

**Comment by muesli** [comment]
@patrick-east Happy to update the PR again with the goimports call you suggested. Let's see.

**Comment by muesli** [comment]
@patrick-east Pushed! How does that look? :smiley:

**Comment by muesli** [comment]
I guess we need a newer (probably even from git master) goimports version for `-format-only`.

**Comment by tsandall** [comment]
@patrick-east what is the right path forward for this PR?

**Comment by patrick-east** [comment]
We need to update the version of goimports we vendor and/or switch to using it in a different way. This will be much easier to do after I get the changes up to move to modules (in progress right now).

After that we re-run it on the current master and commit those changes.

So.. maybe we close this for now and re-open once we have go-imports updated?

**Comment by tsandall** [comment]
OK. I'm going to close this for now. We can revisit once the Go modules stuff is done. When we get around to this change, we'll need to make sure to get it merged ASAP to avoid running into more and more conflicts as other changes are likely to affect imports.

**Comment by patrick-east** [comment]
Opened a new PR for this in #1793

**Pull request closed without merging**